cascades-server/app/src/test/java/pl/gov/coi/cascades/server/TemplateGatewayImplFunctionalIT.java

```diff
@@ -57,6 +60,9 @@ public void setDatabaseIdGateway(TemplateIdGateway templateIdGateway) {
     @Mock
     private Logger logger;
 
+    @Mock
+    private EntityManager entityManager;
```

Comment:
Look at the file name. It should be an integration test.
So we should save entity to database and read it from database.
Don't mock entity manager.